### PR TITLE
Proptypes

### PR DESCRIPTION
**Pull request by snyderc**
Add proptypes to map input. Update field types in node backend.

### CODE REVIEW

File: utils/site-utils.js [hunk 1]
```diff
@@ -8,14 +8,15 @@
...
+ *     latitude: number,
+ *     longitude: number,
  *
  *     // An empty string indicates that there is no availability
  *     // May contain a URL
  *     availability: string,
  *
- *     lastUpdated: ?Date,
+ *     // Number of milliseconds since January 1, 1970
+ *     lastUpdated: number or null,
```
Comment (hblumberg) [inline utils/site-utils.js:19]: Ah thank you for catching this!

File: utils/site-utils.js [hunk 2]
```diff
@@ -8,14 +8,15 @@
  *     // May contain a URL
  *     serves: string,
  * 
- *     latitude: string,
- *     longitude: string,
+ *     latitude: number,
+ *     longitude: number,
```
Comment (hblumberg) [inline utils/site-utils.js:12]: I don't think we parse lat and lng into numbers here, but we could! I believe they're still strings.

**Review by hblumberg — APPROVED** [review]
😍 Thank you so much!!

File: utils/site-utils.js (hunk 2 above)
Comment (snyderc) [reply utils/site-utils.js:12]: When I've checked in the Chrome inspector, they are shown as numbers. Are they stored as strings in Airtable?

**Comment by snyderc** [comment]
Incorporated into https://github.com/codeforboston/vaccinatema/pull/154

**Pull request closed without merging**